vignettes/Multilanguages_comparision.Rmd

```diff
@@ -225,56 +234,55 @@ plot(
   )
 ```
 
-As we can see, models are quite similiar. The biggest difference is residuals distrubution in [0, 0.13] compartment where R `gbm` has biggest mistake ratio. R is worse in compartment [0.13, 0.25] aswell, but diffence is not as easy to spot as in previous area. `CatBoost`, on the other hand, is the best model for residuals [0, 0.13].
+As we can see, models some models differ from each other altough the *java* and *python* models are quite similar.
 
 ## Variable importance

```

Comment:
The graph is illegible, maybe you should take a few variables or change the figure size options.